react/src/utils/helpers.ts

```diff
@@ -61,3 +61,12 @@ export const generateDropappUrl = (
 export const redirectToExternalUrl = (url) => {
   window.location.replace(url);
 };
+
+export const getContrastYIQ = (hex) => {
```

Comment:
Can you add a comment, e.g.
Return true if given RGB hex color is bright, false if dark.
This is achieved by converting the RGB value into YIQ and then checking the Y component (luma). Cf. https://stackoverflow.com/a/946734/3865876

Reply:
Maybe the function name `colorIsBright` would suit better then?